Sloth.Core/Jobs/AggieEnterpriseJournalJob.cs

```diff
@@ -23,8 +24,35 @@ public AggieEnterpriseJournalJob(SlothDbContext context, IAggieEnterpriseService
             _aggieEnterpriseService = aggieEnterpriseService;
         }
 
+        public class AggieEnterpriseJournalJobDetails
+        {
+            public AggieEnterpriseJournalJobDetails()
+            {
+                TransactionRunStatuses = new List<TransactionRunStatus>();
+            }
+
+            public List<TransactionRunStatus> TransactionRunStatuses { get; set; }
+
+            public int TransactionsProcessedCount => TransactionRunStatuses.Count;
+
+            public class TransactionRunStatus
```

Comment:
Did you want this outside the class?

Reply:
I was thinking no, but if you can think of a use case we can extract it.